terraform/monitoring/panels/rds/cpu.libsonnet

```diff
@@ -28,6 +28,10 @@ local targets   = grafana.targets;
       datasource    = ds.cloudwatch,
       namespace     = 'AWS/RDS',
       metricName    = 'CPUUtilization',
+      dimensions  = {
+        DBClusterIdentifier: vars.rds_cluster_id,
+      },
+      matchExact  = true,
       statistic     = 'Average',
```

Comment:
Dis should be max no?

Reply:
https://github.com/WalletConnect/notify-server/pull/330